IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc

```diff
@@ -0,0 +1,74 @@
+ISO-10303-21;
```

Comment:
This file only differs from the file above in GUIDs and some semantic data about file's creation date. I don't see the point in having duplicate files.